### DISCUSSION TITLE
[Scam Alert] Crypto by Alireza علیرضا سبزعلیان

### DISCUSSION THREAD

**Discussion by Blockunity** [Webamoozcom/warning-list#189] [General]
قرار هست درباره کسی صحبت کنیم که خیلی ها احتمالآ میشناسند اش، کسی که به واسطه آشنایی باهاش خیلی ها بچه دار شدند ، ماشین خزیدند و یا به سود رسیدند.

شاید فکر کنید درباره یونیک فایننس فرزین فردین فر میخوایم صحبت کنیم یا پروژه بادران یا حتی امثال علی حسنی ها و باربد معصومی و آرش مارول ها!
خیر
![photo_2024-11-08_22-45-35](https://github.com/user-attachments/assets/8266601c-d296-4841-9c9a-341fb7350a58) 
پ.ن:
![Screenshot 2024-11-08 224843](https://github.com/user-attachments/assets/4123958f-efd1-4777-b863-6faef694d821)
https://x.com/yasharsaljoughi/status/1853433885566087187
https://x.com/yasharsaljoughi/status/1853399356667253115

![photo_2024-11-08_22-45-49](https://github.com/user-attachments/assets/9d65fadb-4073-4cd3-83de-b041477a351a)
![photo_2024-11-08_22-45-59](https://github.com/user-attachments/assets/b530b543-1e86-4ee9-92eb-0cb4ea8279e0)

مخاطب ما علیرضا هست که با اسم crypto by alireza در [تلگرام](https://t.me/cryptobyalireza) شاید بشناسید ولی ما با اسم علی رضا سبزعلیان میشناسیم.
فردی که به واسطه فعالیت در اکوسیستم فانتوم بسیار شناخته شد.

ایشون در گذشته با اموزش و اطلاع رسانی نسبت به بلاکچین شروع به فعالیت کرد و سعی در اگاهی رسانی کرد و حتی در وب آموز نیز ازش نام برده شد اما به مرور زمان بعد از جلب اعتماد افراد مسیر فعالیت اش تغییر کرد و شروع به معرفی پروژه ها و توکن های نا شناخته یا گاها با مارکت کپ پایین یا low cap ها کرد در حالی که خلاف قوانین فعالیتی خودش یعنی توصیه به سرمایه گذاری و پیشنهاد خرید دادن، فعالیت میکنه.

در تلگرام با بررسی کانال تلگرام علیرضا و صحبت با شاکیان ایشون و همچنین اسناد داخل کانال تلگرام متوجه فعالیت مشکوک به دامپ و پامپ* شدیم.
*دامپ و پامپ به زبان سادی یعنی: فرد یا گروهی شروع به خرید ارزی میکنند و بعد از اتمام خرید انرا به کامیونیتی خودشون معرفی میکنند و حجم زیادی پول وارد اون ارز میشه و به اصطلاح پامپ (افزایش) قیمت رخ میده و تیم با سود زیاد شروع به فروش میکنند و بعد قیمت ریزش پیدا میکنه و دامپ رخ میده.
این افراد در گروه و یا کانال هاشون توکن های مختلفی رو معرفی میکنند که گاها بعضی ها **سود ده هستند(برای جلب اعتماد)** تا بتونند به همین واسطه کاربر های جدیدی رو جذب کنند و در میان اون ارز ها شروع به معرفی ارز های خودشون میکنند و دامپ و پامپ میکنند!

چند نمونه از این نوع فعالیت هارو با توجه به اسناد ارائه شده بررسی میکنیم.
**توجه: این اسناد از داخل کانال یا توییتر ایشون آرشیو شدند و اگر مطالب موجود نیستند به این دلیل هست یا ادیت شدند یا پاک شدند توسط علیرضا سبزعلیان.**

نمونه اول جهت جلب اعتماد کاربر توکنی به اسم tomo بود که در مقطعی سود ده و بعد یک ریزش شدید رخ داد و به واسطه سود ده بودند توانست جلب اعتماد کند برای یک طعمه جدید!
یعنی توکن Lua که ۷۴ پست داخل تلگرام از تاریخ 22-02-2021 در توییتر و تلگرام قرار داده شد.
source : https://t.me/cryptobyalireza/279

خیلی از این پست ها ادیت شدند با فاصله یک سال! علت نا مشخص و مطلب اصلی از بین رفته!
![photo_1_2024-11-08_22-56-48](https://github.com/user-attachments/assets/e41f2c61-9344-48bb-a6c4-e9b0628d2ea0)
![photo_5_2024-11-08_22-56-48](https://github.com/user-attachments/assets/ca05bd77-6775-4db7-8825-4a90ce1c84eb)
![photo_6_2024-11-08_22-56-48](https://github.com/user-attachments/assets/c7a865df-09c4-46a8-8322-2510d5560d4f)
![photo_7_2024-11-08_22-56-48](https://github.com/user-attachments/assets/e89cabcb-bbb3-43d4-9485-af0f406f3051)
![photo_8_2024-11-08_22-56-48](https://github.com/user-attachments/assets/7a054edc-5267-415f-8418-bd4aebdc9d79)
![photo_2_2024-11-08_22-56-48](https://github.com/user-attachments/assets/5c8eb924-84d4-436a-8ce9-14aa7df7a16f)
![photo_3_2024-11-08_22-56-48](https://github.com/user-attachments/assets/0b6562d5-40b0-433d-8937-7544921a2853)
![photo_4_2024-11-08_22-56-48](https://github.com/user-attachments/assets/a0ddeca0-13b7-4d16-bbc2-6cf485220d94)

پروژه Lua به شدت توسط ایشون تبلیغ میشد و به صورت مستقیم پیشنهاد سرمایه گذاری میداد و به عنوان یک الماس به مخاطب نشون داده میشد که میتونه مثل tomo سود سنگین بده اما این پروژه در موقعیت هایی که پیشنهاد خرید میشد ریزش شدیدی هم به همراه میداشت که دقیقا دامپ و پامپ میشد.
اما نکته مهم این بود که این پروژه ۳ماه هر یک روز درمیان تبلیغ میشد و یکباره متوقف شد و توکن های ناشناخته جدید جای اون رو میگرفت.
![image](https://github.com/user-attachments/assets/37d86367-01fa-46f2-bfd9-fe08847fdc7f)
https://t.me/cryptobyalireza/1356
![FR0CjyXX0AE6weg](https://github.com/user-attachments/assets/35bbe359-a384-43e1-8a27-8aee7cb1b950)
![FR0CjyXX0AE6weg](https://github.com/user-attachments/assets/7b7a40ee-5aa6-42d6-9fff-d1361021e636)
https://t.me/cryptobyalireza/346
ایشون هر بار توکن های مختلفی رو معرفی میکرد در بازه های کوتاه و بعضی ها سود میدادند بعضی ها ریزش سنگین اما مشکل از اونجایی شروع میشه که نقاط خروج برای ارز ها داده نمیشد و در سقف قیمتی به عنوان کف قیمت معرفی میشد و یا ریزش تبلیغ خرید داده میشد که باز ریزش شدید تر به همراه داشت.
چارت این ارز ها به وضوح زمان پیام های ایشون و دامپ های قیمتی و یا پیشنهاد های خرید رو نشون میده.
![GbdEDlPboAAJZzS](https://github.com/user-attachments/assets/c2af15c0-902a-4b74-8f95-ce941b056fb6)
https://t.me/cryptobyalireza/3537
https://t.me/cryptobyalireza/3860
https://t.me/cryptobyalireza/3878
https://t.me/cryptobyalireza/3888
علیرضا به صورت خیلی حرفه ای با فن بیان ای که داشت پروژه ای که هیچ کاربردی نداشته یا خیلی ساده بوده رو جوری به مخاطب با لغات فنی و بزرگ معرفی میکرده که فرد بهش القا میشده الان با یک ابر پروژه مواجه هست و ترقیب به خرید میشده.
نمونه های زیادی داخل تلگرام ایشون قرار داره از holo 
Mummy , hot, trvl, metis , pivxو ...

اکثر پست های مربوط به این ارز ها با فاصله زیاد ادیت شدند و مشخص نیست چه محتوایی داخلش بوده.
علیرضا با منیپولیت کردن یا به اصطلاح بازی با افکار مخاطب هاش میتونست یک توکن رو که حجم کم معاملات داره رو پامپ کنه و خودش در موقعیت مشخص نقد کنه و باعث ریزش سنگین ارز بشه.
![Gb1d9pOXQAQPmpf](https://github.com/user-attachments/assets/f84b06bc-053b-4f87-990a-f0c0dd623274)

اکثر فعالین این حوزه یا مبتدی ها میدونند ساخت یک توکن کار سختی نیست و زمان زیادی هم نیاز نداره و این نوع افراد خیلی راحت میتوانند در میان لیست ارز هایی که معرفی میکنند توکن های خودشون رو هم به کاربر قالب کنند و از کامیونیتی پول درارند و در اخر کامیونیتی باضرر خارج میشند.
تارگت های بالا برای پروژه ها و ترغیب به خرید در نقاطی که عملا مناسب خرید نیست و فقط باعث ضرر کاربر میشه
مثل تارگت های ۲۰ دلاری برای ارز فانتوم یا ۶۰۰دلاری برای ارز پنکیک یا ۱دلاری برای از Lua و باقی که مشاهده میکنید.

![Gb2tPkHa0AA9NqC](https://github.com/user-attachments/assets/b310902d-c17f-4fc6-8f63-f42c13fae743)
![Gb2tUlWbwAA4ozh](https://github.com/user-attachments/assets/7a064952-c5ca-42a8-b324-2169778f0aee)
![GUJrl9VWEAAh7BU](https://github.com/user-attachments/assets/22439c93-0553-47b2-8126-7453be5d9428)
![FynPRcmWYAAVTG4](https://github.com/user-attachments/assets/a991cdaf-dde3-46ea-80e1-7e7563bf5a6b)
![Fxia-8-aQAAv9-w](https://github.com/user-attachments/assets/6e217255-d825-4fb1-b884-4a2b26d680d5)
![Fxia-80acAQyXlm](https://github.com/user-attachments/assets/6c9b1498-6734-4029-ac70-3609726b1171)
![FSf6Mt_XwAE-1Oz](https://github.com/user-attachments/assets/b3ce2349-e8da-4867-b98a-de73d9ccce82)
https://t.me/cryptobyalireza/1878
![image](https://github.com/user-attachments/assets/101c0320-8beb-47f3-9f84-dd147505f154)
https://t.me/cryptobyalireza/4149
چند روز گذشته کاربری متوجه میشه فاندر یکی از پروژه هایی که علیرضا به مدتی طولانی در حال تبلیغ آن هست ایرانی هستند و این باعث میشه که
 ما نسبت به دیتای ارسال شده تحقیق کنیم.

از سال 2023 علیرضا داخل کانال تلگرام اش شروع به معرفی پروژه ای به اسم wigoswap میکنه.
این پروژه در اصل در سال ۲۰۲۱ با توجه به کد های [گیت هاب](https://github.com/wigoswap) اش راه اندازی میشه و توکن اش در سال ۲۰۲۲ عرضه میشه که در اوایل با بازخورد خوب مواجه میشه و بعد ریزش به شدت شدیدی پیدا میکنه و دیگه حجم معاملات زیادی رو جذب نمیکنه و عملا تبدیل به یک پروژه مرده میشه تا حدی که بیش از ۳ سال کد های پروژه هیچ تغییری نداشتند و اعضای تیم پروژه عملا هیچ فعالیتی در گیت هاب نداشتند‌.
با توجه به این اطلاعات شروع به بررسی کردیم و به مطالب زیر رسیدیم:

با بررسی اولیه روی گیت هاب این پروژه و چک کردن git log کد ها متوجه شدیم در تاریخ 29/12/2021 فردی به اسم a. Sabzalian اولین مطلب درباره ویگو رو نوشته یا به اصطلاح فنی first commit این پروژه رو زده یعنی این فرد اندر و سازنده این پروژه هست.
![image](https://github.com/user-attachments/assets/5ebd5958-ab3e-48a0-9fbd-d16d4f9de155)
https://archive.softwareheritage.org/browse/snapshot/51aeb20ce7ae4e9a1987a304aaeb343800e3e241/log/?origin_url=https://github.com/wigoswap/wigo-core
با بررسی اسم a sabazlian به سایپی به اسم [اسپارد](https://espard.com/%D8%A7%D8%B3%D8%AA%D8%A7%D8%B1%D8%AA%D8%A7%D9%BE-%D9%86%D8%B8%D8%A7%D9%81%D8%AA-%D9%88-%D8%AE%D8%AF%D9%85%D8%A7%D8%AA-%D9%85%D9%86%D8%B2%D9%84-%D8%A7%D8%B3%D9%BE%D8%A7%D8%B1%D8%AF) رسیدیم که ایشون با اسم کامل علیرضا سبزعلیان به عنوان اعضای این شرکت هستند که با 
![GbXDOL7X0AApdMy](https://github.com/user-attachments/assets/814617fa-66fe-4e2d-9503-194c5ab6cdca)
پرسوجو هم متوجه شدیم این شرکت شاکی هایی داشتند نسبت به پرداخت حقوق ها!
![image](https://github.com/user-attachments/assets/3252c6ff-e333-4ef2-85b4-659fbdd84626)
![image](https://github.com/user-attachments/assets/5299baff-2a53-48da-8c74-95e9e5e4cd44)

"متاسفانه مطلب برای سال ۱۳۹۶ هست و در حال حاضر امکان دسترسی به متن شکایت نیست اما شاکی شرکت با ما ارتباط گرفت و موضوع رو با ما در جریان گذاشت
صحت این ادعا با دریافت بخشی از شکایت نامه به درخواست ما میتونه تایید بشه که فعالا بخاطر مهاجرت فرد امکان دسترسی به شکایت نامه نداریم."

با بررسی بیشتر و عکس های این فرد هویت ایشون تایید میشه.
ادعای مطرح شده از طرف یکی از اعضا این بود که طبق بررسی تاریخچه حساب فاندر پروژه یعنی اقای [AaronL3b](https://x.com/AaronL3b) متوجه میشه که در تاریخ قبل از ۲۰۲۲ ایشون به صورت فارسی داخل توییتر پست میگذاشتند.

![GbXAFpCXMAAXh1f](https://github.com/user-attachments/assets/667f08c7-0907-4e0f-b02e-531df48e6790)
![GbiKlDLawAA52Eu](https://github.com/user-attachments/assets/c635b85e-66cc-442b-8fe8-26adc59df560)
با بررسی رپلای ها محتوای ارایه شده حدس زده میشه که برای یک خانم بوده که پست های سکسی منتشر میکرده!

در تاریخ 2024 فاندر پروژه اقای aaron مدعی میشند داخل دیسکورد که حساب رو خریدند اما ادعا ایشون کاملا کذب هست!
https://discord.com/channels/937634891904978974/940577075574747156/1304080720469299322
![photo_2024-11-08_23-56-50](https://github.com/user-attachments/assets/77d39643-6bf9-465c-b894-ca7469312f8a)
به چند دلیل منطقی:
**ا. فردی خارجی که قصد راه اندازی پروژه ای رو داره یک حساب زیر چند هزار فالوور خریداری نمیکنه!
۲. فرد خارجی برای یک پروژه خارجی حساب توییتر یک ایرانی اون هم با محتوای سکسی رو خریداری نمیکنه!**
پس این ادعا از دم رد میشه.
نکته جالب تر اینجا هست که با توجه به نوع ادبیات پست های aaron و علیرضا و شباهت زیاد سبک پیام ها و ثابت میشه کرد که آرون در اصل خود علیرضا هست و احتمالا اون حساب توییتری متعلق به خود ایشون بوده و بعد از ۲ سال فعالیت نکردند و پاک کردن پیام ها شروع به پست های ا
نگلیسی با اسم جدید میکنه.
![GbjN7riX0AAcfTS](https://github.com/user-attachments/assets/8c7ab1fb-6d22-4645-a787-4b614d7bee4c)
![GbjN7rnWwAAU-Th](https://github.com/user-attachments/assets/57ef4d07-2963-4580-99b4-a7b11da8519e)

و از طرف دیگه چون علیرضا سبزعلیان به عنوان author یا نویسنده first commit پروژه ویگو رو زده تمام اینها ثابت میکنه که این دو فرد در اصل یکی هستند.
![Gbevm25agAA24rb](https://github.com/user-attachments/assets/be306455-f7ca-4f60-b4e9-3c956eab1b0c)

بعد از منتشر شدن اینکه پشت پروژه ویگو یک ایرانی به اسم علیرضا سبزعلیان هست و منتشر شدن اطلاعات گیت هاب بلافاصله تمام مطالب داخل گیت هاب که امکان دیده شدن اسم علیرضا رو داشته یکباره پاک میشه و دوباره بعد از چند ساعت اضافه میشه!
![GbYiupwWIAAHaDf](https://github.com/user-attachments/assets/27aec719-0368-47d6-89f0-307e8eb540fc)
![GbYiu8sWQAAzdLN](https://github.com/user-attachments/assets/2c4c36f2-bb4e-410d-a901-8750f94ba55b)
https://x.com/Marsha1am/status/1852685997135294656
https://x.com/Marsha1am/status/1852706490286473635
گیت هابی که بیش از ۲ سال هیچ فعالیتی نداشته و دقیقا همون مطالب تعویض میشه!
بلافاصله داخل توییتر علیرضا مدعی میشه که در این پروژه نقشی نداشته و به عنوان contributor یا مشارکت کننده فعالیت داشته و عملا اسم ایشون رو به واسطه یک فورک و بدون اجازه ایشون داخل گیت هاب قرار دادند.
https://t.me/cryptobyalireza/4285
![GbZvaJQaEAEEalY](https://github.com/user-attachments/assets/1f2688bb-afeb-4521-993c-9a01e76d8f5c)
این ادعا از پایه بی اساس و رد میشه:
**علت:** داخل گیتهاب اسم علیرضا یه عنوان author یا نوسینده یا سازنده نوشته شده پس ادعای contributor رد میشه و همچنین اگر فورکی داخل گیت هاب این پروژه ثبت بشه داخل تاریخچه فعالیت نیز ثبت میشه که همین هم رد میشه چون از قبل first commit که توسط ایشون نوشته شده هیچ دیتا و تاریخچه ای نیست!
`سوالی که بی پاسخ موند و علیرضا از پاسخ بهش بی اعتنایی کرد این بود که با توجه به این دیتا ها که به وضوح نمایان گر این هست که خود ایشون فاندر هست چرا اطلاعات غلط به کامیونیتی ارائه داد و چرا با توجه به ادعا هاش که این پروژه ویگو یک الماس هست و بسیار قوی هست بهش افتخار نکرد و نگفت برای ایشون هست!؟ چرا دروغ گفت به مخاطب ها؟ یا اینکه این پروژه ایراد های پنهان دارد که اگر فردای روز مثل پروژه های دیگه ای که معرفی کرد و شکست خوردند این هم شکست خورد بتونه سلب مسئولیت کنه!
`
در نتیجه با توجه شبه ما نسبت به Aaron رفع شد و با قطعيت مشخص شد که خود علیرضا هست.
در ادمه بخاطر رفتار عجیب و دستپاچه ای علیرضا باعث شد که بیشتر روی این پروژه وارد بشیم.

طبق بررسی اولیه در سایت های بررسی کننده کانترکت متوجه شدیم که امتیاز بسیار پایینی این پروژه داره.
https://tokensniffer.com/token/ftm/aecdhhnja9ojkkd1hjzg3zssqvhu0ks690cqcvzpdmjjmzx50i36beg1jp2k
![image](https://github.com/user-attachments/assets/5193c1d4-0209-483f-abc0-8a7aaffd9deb)
https://dexscreener.com/fantom/0xb66e5c89eba830b31b3ddcc468dd50b3256737c5
![image](https://github.com/user-attachments/assets/c8d49b5b-c451-4978-93f2-12a9d04e96eb)
https://de.fi/scanner/contract/0xe992beab6659bff447893641a378fbbf031c5bd6?chainId=ftm
![image](https://github.com/user-attachments/assets/91eab0c1-a4bc-49ef-bd53-01f6d36eb761)

ادامه مطلب :

https://github.com/Blockunity/Wanted-List/discussions/5

**Comment by My1Coin** [comment]
این رفتار خطرناک علیرضا سبزعلیان قرار نیست تموم بشه؟؟
اینکه بیاد پروژه هایی که در اوایل کارش معرفی کرده و سودده بودند رو با پروژه های الان خودش مقایسه کنه
🧵
Image

مثلا ارجاع بده به پست های معرفی متیک در سال ۲۰۲۱ و کاربر تازه کاری که آشنایی چندانی با فضای کریپتو نداره فکر می‌کنه آشغال هایی که الان معرفی می‌کنه هم قراره همون مسیر رو بروند
در صورتیکه ۹۵٪ پروژه هایی که در ۲سال گذشته در کانالش معرفی کرده از قبل سقف خودشون رو تجربه کرده بودند
Image

Image
بطور مثال پروژه مومیایی در ۲۳ فوریه ۲۰۲۳(بهمن ۱۴۰۲) در قیمت ۸ دلار ‌به عنوان یک پروژه فوق العاده و درآمدزا معرفی شده
توکنی که از همونجای ریزش سنگینی کرد و در حال حاضر ۲۳ سنت قیمت دارد
یعنی اگر شما در روز معرفی ۱۰۰۰ دلار از این توکن خریداری کرده بودی، الان تبدیل به ۲۷ دلار شده بود
یا توکن Luaswap که در ۰.۵ دلار این توکن را مجانی میخواند و توصیه می‌کرد کسانی که در این قیمت فروشنده این توکن هستند شدیدا ضرر خواهند کرد و پشیمان می‌شوند.
شاید بگید خیلیا توکن معرفی کردند و بعد از مدت ها قیمت این توکن ها سقوط کرده
Image

هرکسی توکنی معرفی کرد و در سقف قیمتی همچنان توصیه به خرید اون توکن داشت، از ۲ حالت خارج نیست
یا در اون توکن ذینفع محسوب میشه یا بابت تبلیغاتش پول گرفته
وگرنه کسی که ریگی به کفشش نباشه
وقتی قیمت یک توکن یک پنجم شده، حداقل کاری که می‌تونه بکنه اینه که فرمان خروج بده و عذرخواهی کنه
Image
ولی متاسفانه علیرضا نه تنها در هیچ کدام از پروموت کردن ها، توصیه ای به فروش توکن و نقد کردن سود نداده، بلکه با تارگت های رویایی مثل لواسواپ ۱.۵ دلاری، مومیایی ۱۰۰×، ویگو سواپ ۴ دلاری، کیک ۶۰۰ دلاری، متیس هزار دلاری و... دنبال کنندگان خود را فومو کرده و به چاله متضرر شدن هل داده
Image
در پایان باید گفت وظیفه امثال من و شما در قبال چنین افرادی که همواره سرمایه دنبال کنندگان خود را به خطر انداخته و با اغوا و فریب به فکر دوشیدن آنها هستند، این است که آنها را افشا کنیم تا دیگران در تله و دام این قبیل افراد نیافتند
و توجهی به توجیه و سمپاشی دیگرانی که از قبل فریبِ افرادی مثل علیرضا را خورده و در دام آنها افتاده اند، نکنیم
چون آنها دقیقا مشابه فردی هستند که در چاه افتاده و با حیله قصد دارند شما را هم وارد همان چاه کنند تا سوار بر شانه‌های شما شده و از چاهی که در آن هستند به سلامت خارج شوند

تصاویر در توییت زیر قابل مشاهده است

https://threadreaderapp.com/thread/1856623665204408322.html

منابعی که در این رشته توییت استفاده شده اند همگی از کانال تلگرام علیرضا سبزعلیان alireza.s با آیدی cryptobyalireza برداشته شده اند 

پست ارجاع به متیک 
https://t.me/cryptobyalireza/4370

پست معرفی توکن پروژه مومیایی $MUMMY در ۲۳ فوریه ۲۰۲۳
https://t.me/cryptobyalireza/2067

چارت توکن مامی در دکس اسکرینر
https://dexscreener.com/fantom/0x2A6538a456650Cd454Dcd8f0B4665183DBa0bb27

نمونه ای از پست های تشویق به خرید و هولد توکن پروژه LuaSwap که هم اکنون قابل فروش نیست
https://t.me/cryptobyalireza/540
https://t.me/cryptobyalireza/330
https://t.me/cryptobyalireza/346
https://t.me/cryptobyalireza/1262

نمونه ای از پست تارگت کیک ۶۰۰ دلاری، متیس هزار دلاری و فانتوم ۳۰ دلاری 
https://t.me/cryptobyalireza/4149

نمونه ای از تارگت صد برابر شدن  قیمت توکن مومیایی
$MUMMY 100X
https://t.me/cryptobyalireza/2148

نمونه ای از تارگت ۳-۴ دلاری برای توکن ویگو سواپ WiGo 
https://t.me/cryptobyalireza/3309
